postgres/pgdump.go

```diff
@@ -620,6 +659,19 @@ func toSchemaKeys(conv *internal.Conv, table string, s []string) (l []schema.Key
 	return l
 }
 
+// toIndexKeys converts a string list of PostgreSQL primary keys to
+// schema Index keys.
+func toIndexKeys(conv *internal.Conv, table string, s []nodes.Node) (l []schema.Key) {
+	for _, k := range s {
```

Comment:
not sure what do k and s stand here for - why not something like `for _, node := range nodes`

Reply:
as nodes is an imported library name, we can not use it here.